### PR TITLE
REFACTOR/TEST: let e2e tests run in both mobile and desktop view

### PR DESCRIPTION
**Pull request by RGPosadas**
This PR aims to have the E2E tests run on both mobile and desktop view. It was previously running at a random window size, and since our app is a PWA, both views must be covered.

You may see how our sole E2E test runs [here](https://github.com/RGPosadas/Mull/wiki/Acceptance-Test-GIFs).

### CODE REVIEW

File: apps/mull-ui-e2e/src/integration/create-event.spec.ts [hunk 1]
```diff
@@ -1,85 +1,98 @@
...
-    );
+const frameSizes = [
+  [375, 812],
+  [1920, 1080],
+];
+
+frameSizes.forEach((size) => {
+  describe('US-1.1: Create Events', () => {
+    beforeEach(() => {
+      cy.viewport(size[0], size[1]);
```
Comment (RGPosadas) [inline apps/mull-ui-e2e/src/integration/create-event.spec.ts]: FYI: The changes I added are lines 3-6, 8, and 11. The rest of the lines are changed due to tabs.

**Comment by osahtout** [comment]
I forgot to commit the fix for a styling issue in my pull request #60 so I'll include it in this pull request. Please note the changes are unrelated to any changes in this pull request.

**Review by cristian-aldea — APPROVED** [review]

File: apps/mull-ui-e2e/src/integration/create-event.spec.ts [hunk 2]
```diff
@@ -1,85 +1,98 @@
...
-    );
+const frameSizes = [
+  { res: [375, 812], name: 'mobile' },
+  { res: [1920, 1080], name: 'desktop' },
+];
+
+frameSizes.forEach((frame) => {
+  describe(`US-1.1: Create Events (${frame.name} view)`, () => {
+    beforeEach(() => {
+      cy.viewport(frame.res[0], frame.res[1]);
```
Comment (RGPosadas) [inline apps/mull-ui-e2e/src/integration/create-event.spec.ts:11]: FYI: The changes I added are lines 3-11. The rest of the lines are changed due to tabs.

**Review by cindyslittleplanet — APPROVED** [review]
LGTM!

**Comment by RGPosadas** [comment]
The SonarCloud analysis will be ignored for the pull request because the report isn't accurate. The code it reports as uncovered was not modified in this pull request. 

The code introduced and modified are fully covered by the tests. 
image

**Pull request merged**